contrib/scoverage/src/mill/contrib/scoverage/ScoverageModule.scala

```diff
@@ -189,18 +183,18 @@ trait ScoverageModule extends ScalaModule { outer: ScalaModule =>
     }
 
     override def compileResources: T[Seq[PathRef]] = outer.compileResources
-    override def generatedSources: Target[Seq[PathRef]] = T { outer.generatedSources() }
-    override def allSources: Target[Seq[PathRef]] = T { outer.allSources() }
+    override def generatedSources: T[Seq[PathRef]] = outer.generatedSources
```

Comment:
I must admit I added those wrappers on purpose in the past, in the believe they add more meaningful context to the targets, which might be useful when debugging. But I never verified, whether this is needed or useful. I see your point with this cleanup, as after your change we produce less byte code and safe some cycles at runtime.